fury/ui.py

```diff
@@ -3504,14 +3505,174 @@ def __init__(self, labels, checked_labels, padding=1, font_size=18,
                                           checked_labels=checked_labels)
 
     def _handle_option_change(self, option):
-        for option_ in self.options:
+        for option_ in self.options.values():
             option_.deselect()
 
         option.select()
         self.checked_labels = [option.label]
         self.on_change(self)
 
 
+class ScrollBar(UI):
+
+    def __init__(self, ListBox2D):
+        self.listbox = ListBox2D
+        self.reverse_scrolling = self.listbox.reverse_scrolling
+
+        denom = len(self.listbox.values) - self.listbox.nb_slots
+        if not denom:
+            denom += 1
+        super(ScrollBar, self).__init__()
+        self.scroll_step_size = (self.listbox.slot_height *
+                                 self.listbox.nb_slots -
+                                 self.scroll_bar.height) / denom
+        self.scroll_bar_active_color = self.listbox.scroll_bar_active_color
```

Comment:
`self.scroll_bar_active_color` is really repetitive. We already know that we are on the scrollbar class. Can you change the variable name? Just call it `active_color`.

Same comment for all variables above